tutorials/tour/_posts/2017-02-13-unified-types.md

```diff
@@ -11,15 +11,15 @@ next-page: classes
 previous-page: basics
 ---
 
-In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates the class hierarchy.
+In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates a subset the class hierarchy.
```

Comment:
I don't think it's pedantic … `Seq` is a type constructor, not a type. You can't have a variable of type `Seq`. You're right of course that we could say `Seq[_]` but that's shorthand for `Seq[A] forSome { type A }` which I think raises more questions than it answers.